regions/core/core.py

```diff
@@ -283,47 +259,29 @@ def union(self, other):
         from .compound import CompoundSkyRegion
         return CompoundSkyRegion(self, other, operator.or_)
 
-    @abc.abstractmethod
-    def contains(self, skycoord):
+    def contains(self, skycoord, wcs):
         """
-        Checks whether a position or positions fall inside the region.
+        Check whether a sky coordinate falls inside the region
 
         Parameters
         ----------
         skycoord : `~astropy.coordinates.SkyCoord`
             The position or positions to check
+        wcs : `~astropy.wcs.WCS` instance
+            The world coordinate system transformation to assume
         """
+        from .pixcoord import PixCoord
```

Comment:
Ah, OK, I see we have `from .core import _DEFAULT_WCS_ORIGIN, _DEFAULT_WCS_MODE` in `pixcoord.py`.  This is not nice that we have circular imports between `pixcoord.py` and `core.py`. I'm +1 to move those constants to `pixcoord.py` to have it be standalone (and because all pixel / world transforms should go through `PixCoord` to abstract away this choice we've made for `regions`.